9_collaborator_data/src/write_functions.R

```diff
@@ -46,3 +46,18 @@ save_dist_matrices <- function(dist_mat_ind, out_ind) {
 
 
 }
+
+write_zarr_tarfile <- function(dat_ind, out_ind){
+  out_file <- as_data_file(out_ind)
+  sys <- reticulate::import('sys', convert=FALSE)
+  position <- 0
+  position_int = as.integer(position)
+  sys$path$insert(position_int, '9_collaborator_data/src/')
+  write_py <- reticulate::import('write_functions_py')
```

Comment:
seems `write_functions_py.py` either needs to be a depends of the target or an input of this function. Otherwise changes to this file won't be tracked. 

Cool R/py integration!

Reply:
Ah right. Good point.

Reply:
because I am desperate to write a little code today 🤪 

```r
dirname('9_collaborator_data/src/write_functions_py.py')
[1] "9_collaborator_data/src"
tools::file_path_sans_ext(basename('9_collaborator_data/src/write_functions_py.py'))
[1] "write_functions_py"
```

Reply:
instead of using `sys` and inserting path to the python module, could you use: 
>write_py <- reticulate::import_from_path('write_functions_py', path = '9_collaborator_data/src/')

Reply:
much cleaner. Thanks, Jake. I didn't know about the `path` arg